CHANGES.md

```diff
@@ -1,3 +1,9 @@
+2018-09-11, Version 1.12.2
+==========================
+
+ * Allow wsdl's to be loaded from cache in a sync way (Dom Storey)
+
+
```

Comment:
No need to update this changelog manually. Our release process will automatically update it using information from the commits.